src/Options/Definitions.js

```diff
@@ -28,6 +28,12 @@ module.exports.ParseServerOptions = {
     help: 'Add headers to Access-Control-Allow-Headers',
     action: parsers.arrayParser,
   },
+  allowExpiredAuthDataToken: {
+    env: 'PARSE_SERVER_ALLOW_OLD_AUTH_DATA_TOKEN',
```

Comment:
This still says "OLD" token in `env`. Please run `npm run definitions` to make sure this is corrected automatically.